journals/apps/search/backend.py

```diff
@@ -78,17 +84,11 @@ def put(self):
 
         # TODO is this the right place? Only do it once if it doesn't exist
         if self.name == JOURNAL_DOCUMENT_INDEX_NAME:
-            body = {'description': 'Extract attachment information',
```

Comment:
I think we should remove the `put` override here and create a method which creates ingest attachment pipeline if one does not exists and call that method in `add_item` and `add_items` methods.